src/store/cache/collections/sagas.js

```diff
@@ -147,7 +147,8 @@ function* confirmCreatePlaylist(uid, userId, formFields, source) {
         // If the user is trying to upload artwork, check that an artwork gets set.
         if (formFields.artwork && formFields.artwork.file) {
           check = playlist =>
-            some([playlist], toConfirm) && playlist.cover_art_sizes
+            some([playlist], toConfirm) &&
+            playlist.playlist_image_sizes_multihash
```

Comment:
don't we massage out this field into cover_art_size? maybe add a comment if im missing something

Reply:
discussed on slack